### PR TITLE
frontend: PluginSettings: Use author for plugin origin

### PR DESCRIPTION
**Pull request by skoeva**
This change makes the formatting of the plugin names consistent, using the plugin author for the origin value. When the author is not set, the default origin is "Unknown".

Fixes: #2544

### Testing
- [X] Open Headlamp and navigate to the plugin settings
- [X] Note the consistent plugin name formats and author listed under the origin column

image
image

### CODE REVIEW

**Comment by illume** [comment]
Can you please split the commits to changes in separate components out?
(Maybe it’s just the subtitle change that could be put in its own commit… but there’s lots of files changed so I could have missed something else.)

**Comment by illume** [comment]
I wonder if we can use the fit repo if it’s there instead of unknown?
Also maybe that should even be displayed… because it can be more useful than whatever the author is put there.

**Review by joaquimrocha — DISMISSED** [review]

File: frontend/src/components/App/PluginSettings/PluginSettingsDetails.tsx [hunk 1]
```diff
@@ -107,6 +107,9 @@ export function PluginSettingsDetailsPure(props: PluginSettingsDetailsPureProps)
   const [enableSaveButton, setEnableSaveButton] = useState(false);
   const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
   const history = useHistory();
+  const [author, name] = plugin.name.includes('@')
+    ? [plugin.name.substring(1).split(/\/(.+)/)[0], plugin.name.split(/\/(.+)/)[1]]
```
[the remark this answers is not in the document]
Comment (skoeva) [reply frontend/src/components/App/PluginSettings/PluginSettingsDetails.tsx]: fixed

**Comment by skoeva** [comment]
@illume added a separate commit for the subtitle change, lmk if this should be split up further to reflect the change itself vs the context it's being used in

**Review by illume — APPROVED** [review]
👍 thanks

**Pull request merged**